src/adapter/src/command.rs

```diff
@@ -221,6 +221,15 @@ pub enum Command {
         conn_id: ConnectionId,
         tx: oneshot::Sender<Result<ExecuteResponse, AdapterError>>,
     },
+
+    /// Execute a side-effecting function from the frontend peek path.
```

Comment:
IMO, we don't have to mention here that it's from the frontend peek path, eventually everything will come "from the frontend". But maybe you have plans to clean this up holistically already once the work is done